docs/setup/main_setup.md

```diff
@@ -71,3 +72,13 @@ To not be forced to reload the scene in Webots when rebuilding the controller, y
 ```sh
 ./pepsi run
 ```
+
+## Running Behavior Simulator
+
+To be able to run the current behavior simulator files, you have to install lua ``ìnspect``` package, either by downloading and saving it to the lua path (e.g., you hulk repo) or by using a lua package manager.
```

Comment:
The \` around `inspect` seem unbalanced. Try just one on each side.

Reply:
Done